CogniteSdk/test/fsharp/Common.fs

```diff
@@ -19,13 +19,13 @@ module Common =
 
     let readClient =
         createClient
-            (Environment.GetEnvironmentVariable "TEST_API_KEY_READ")
+            (Environment.GetEnvironmentVariable "FUSION_DOTNET_TEST_API_KEY_READ")
             "publicdata"
             "https://api.cognitedata.com"
 
     let writeClient =
         createClient
-            (Environment.GetEnvironmentVariable "TEST_API_KEY_WRITE")
+            (Environment.GetEnvironmentVariable "FUSION_DOTNET_TEST_API_KEY_WRITE")
```

Comment:
not sure we should change those, might be some people working only with sdk, but not with fusion?

Reply:
We can do this but then we need to update the keys used for Github Actions

Reply:
But it will break the build now if we do this. Let's take it another time ....

Reply:
Done 👍